test/models/reminder_test.rb

```diff
@@ -36,4 +36,12 @@ def setup
     Reminder.conditionally_create_reminders
     assert_equal 10, Reminder.count
   end
+
+  test 'reminders can only be created with internal volunteers' do
+    assignment = create :assignment
+    assignment.volunteer.update(external: true)
+    assignment.reload
+    reminder = build :reminder, volunteer: assignment.volunteer
+    refute reminder.valid?
```

Comment:
You should also test for specific errors, something like `assert reminder.errors[:volunteer].present?`